presto-iceberg/src/test/java/com/facebook/presto/iceberg/IcebergDistributedTestCase.java

```diff
@@ -25,12 +25,12 @@
 import static java.util.Objects.requireNonNull;
 import static org.testng.Assert.assertTrue;
 
-public class IcebergDistributedTestBase
+public class IcebergDistributedTestCase
```

Comment:
Why rename?  In Presto, IcebergDistributedTestBase has 3 subclasses, and that's why it's named as "Base". Can you please revert this change?

Reply:
I do not remember why I changed the class name, it was probably some suggestion from IDE which got implemented by mistake. Anyways I reverted this.